tools/python/boutiques/localExec.py

```diff
@@ -34,46 +34,45 @@ def __init__(self,desc,options={}):
     # The set of output parameters from the json descriptor
     self.outputs     = self.desc_dict['output-files'] # Structure: [ {id:...},...,{id:...} ]
     # The set of parameter groups, according to the json descriptor
-    self.groups     = self.desc_dict['groups'] if 'groups' in self.desc_dict.keys() else []
+    self.groups     = self.desc_dict['groups'] if 'groups' in list(self.desc_dict.keys()) else []
```

Comment:
not sure why this list and the two other ones below has to be created but fair enough.